MoleQla/Models/seccion.py

```diff
@@ -1,34 +1,18 @@
 # -*- encoding: utf-8 -*-
-from openerp.osv import fields, osv
+from openerp import fields, models, api
 
-class seccion(osv.osv):
+class seccion(models.Model):
     
     _name = "seccion"
     _description = "Seccion"
     
-    _columns = {
-        'nombre' : fields.char('Nombre', size=128),
-        'descripcion': fields.char('Descripción', size=128),
-        'max_articulos': fields.integer('Número máximo de artículos por número'),
-        'editor':fields.one2many('editor', 'seccion_id','Editor'), 
-        'maquetador':fields.one2many('maquetador', 'seccion_id','Maquetador'),
-        }
-                            
-    _defaults = {
-        'max_articulos' : 3, 
-        }
     
-    def name_get(self, cr, uid, ids, context=None):
-        
-        if context is None:
-            context = {}
-        res = []
-        
-        for record in self.browse(cr, 1, ids, context=context):
-            seccion_name = record.nombre
-            
-            
-            res.append((record.id, seccion_name))
-        return res
+    nombre = fields.Char('Nombreeeeee', size=128)
```

Comment:
You can rename this field to "name" so that Odoo will know that it's the record's display name. Alternatively you can define a new compute field "display_name" which will return "nombre" (see articulo model): http://odoo-new-api-guide-line.readthedocs.io/en/latest/environment.html#displayed-name-of-record
As it stands now seccion records will not be displayed properly